packages/common/src/models/User.ts

```diff
@@ -100,3 +109,54 @@ export type UserMultihash = Pick<
   User,
   'metadata_multihash' | 'creator_node_endpoint'
 >
+
+/** Converts a SDK `full.UserFull` response to a UserMetadata. Note: Will _not_ include the "current user" fields as those aren't returned by the Users API */
+export const userMetadataFromSDK = (
```

Comment:
Open to debate on this pattern. I'm considering just writing zod schemas and using those to parse here instead.
The goal is to transform a SDK object into the model we have here so that calls to a migrated endpoint return the exact same shape as the previous AudiusAPIClient call. We can do another pass later to decide if we want to start using SDK types directly.

Reply:
I started with that approach and abandoned it for a few reasons:

1. Input to response adapters requires objects that are the same shape as the models defined for APIClient, which differs from what is returned by SDK (mostly do to the undefined vs. null issue).
2. The ResponseAdapter has logic to default initialize number fields, which the SDK types say will always have valid values. So I think this caught some unnecessary logic in our response adapters.
3. I was hoping that part of the SDK migration would be removing ResponseAdapters in favor of the model classes having local functions meant to initialize an instance.

Reply:
Fair enough! I think #3 is something I would generally agree with, but that will also be a decent tack on of work